app/models/chargeback.rb

```diff
@@ -21,6 +21,7 @@ def self.build_results_for_report_chargeback(options)
     rate_cols = ChargebackRate.where(:default => true).flat_map do |rate|
       rate.chargeback_rate_details.map(&:metric).select { |metric| perf_cols.include?(metric.to_s) }
     end
+    rate_cols.map! { |x| (x == "v_derived_cpu_total_cores_used") ? "cpu_usage_rate_average" : x }
```

Comment:
> @zeari Was this your intension ? Briefly said : calculate 'Used CPU Cores' based on 
> v_derived_cpu_total_cores_used instead of unrelated metric cpu_usage_rate_average

Yes. `v_derived_cpu_total_cores_used = cpu_usage_rate_average *   (cpu_usage_rate_average * derived_vm_numvcpus) / 100.0`
So in that select we need to fetch `cpu_usage_rate_average` and `derived_vm_numvcpus`
(but `derived_vm_numvcpus` is already included)

This is just a way to deal with that virtual column.